packages/react/src/components/TextInput/PasswordInput.tsx

```diff
@@ -442,6 +444,8 @@ PasswordInput.propTypes = {
    */
   invalid: PropTypes.bool,
 
+  readOnly: PropTypes.bool,
```

Comment:
missing comment here
```suggestion
   /**
   * Whether the PasswordInput should be read-only
   */
  readOnly: PropTypes.bool,
```